models/apps/rancher-gatekeeper-operator.js

```diff
@@ -1,10 +1,18 @@
 import { MODE, _CREATE } from '@/config/query-params';
 import { addParams } from '@/utils/url';
 import { GATEKEEPER_CONSTRAINT_TEMPLATE } from '@/config/types';
+import { downloadFile } from '@/utils/download';
 
 export default {
   availableActions() {
     let out = this._availableActions;
+    const downloadAction = {
```

Comment:
A regular resource-instance has a download action already, so why is it missing here to need to be re-added?

Reply:
I had initial thought I may change the label which is why I included the whole object. I'll change that. But I do have to change the enabled check here because the check is not reliant on the `view` link. I'll fix it so its only changing that flag and not including the whole object.

Reply:
Ok that makes more sense with a different download action now.

It should probably be giving you the whole app definition instead of a subset of its config, but that doesn't really start to make sense until you can manage helm apps To import it, so this seems ok as something yaml-ey for now.

Reply:
Updated to just change the action enabled flag.